rust/headless-client/src/windows/wintun_install.rs

```diff
@@ -41,7 +41,7 @@ pub(crate) fn ensure_dll() -> Result<PathBuf, Error> {
     let dir = path.parent().ok_or(Error::DllPathInvalid)?;
     std::fs::create_dir_all(dir).map_err(|_| Error::CreateDirAll)?;
 
-    tracing::info!(?path, "wintun.dll path");
+    tracing::debug!(?path, "wintun.dll path");
```

Comment:
Not related to the PR but I just noticed I haven't needed this log line in a while